draft-ysl-cats-metric-definition.md

```diff
@@ -70,15 +70,11 @@ This document defines a set of computing metrics used for Computing-Aware Traffi
+Service providers are deploying computing capabilities across the network for hosting applications such as distributed AI workloads, AR/VR and driverless vehicles, among others. In these deployments, multiple service instances are replicated across various sites to ensure sufficient capacity for maintaining the required Quality of Experience (QoE) expected by the application.  To support the selection of these instances, a framework called Computing-Aware Traffic Steering (CATS) is introduced in {{!I-D.ietf-cats-framework}}.
 
-CATS is a traffic engineering approach that takes into account the dynamic nature of computing resources and network state to optimize service-specific traffic forwarding towards a given service contact instance{{!I-D.ietf-cats-framework}}. Various metrics may be used to enforce such computing-aware traffic steering policies.
+CATS is a traffic engineering approach that optimizes the steering of traffic to a given service instance by considering the dynamic nature of computing and network resources. To achieve this, CATS components (C-PS, C-Forwarders, etc.) require performance metrics for both communication and compute resources. Since these resources are deployed by multiple providers, standardized metrics are essential to ensure interoperability and enable precise traffic steering decisions, thereby optimizing resource utilization and enhancing overall system performance.
 
-To steer traffic to a service contact instance, CATS components(C-PS, C-Forwarders, etc.) need information of the service instance's computing status.  In addition to network-related metrics, a common definition of relevant computing metrics is essential for effective coordination between network devices and compute instances. Standardized metrics enable precise traffic steering decisions that optimize resource utilization and improve overall system performance.
-
-Various considerations for metric definition are proposed in {{?I-D.du-cats-computing-modeling-description}}, which are useful in defining computing metrics.
-
-Based on the considerations defined in {{?I-D.du-cats-computing-modeling-description}}, this document defines relevant computing metrics for CATS by categorizing the metrics into three levels based on their complexity and granularity details.
+Various considerations for metric definition are proposed in {{?I-D.du-cats-computing-modeling-description}}, which are useful for defining computing metrics. This document categorizes the relevant computing metrics for CATS into three levels based on their complexity and granularity, following the considerations outlined in {{?I-D.du-cats-computing-modeling-description}}.
```

Comment:
Maybe it should be "This document categorizes the relevant computing metrics as well as network metrics for CATS..."

Reply:
I am not sure I fully understand this comment. I think the approach (also from what the chairs usually want us to focus on) is that we focus on CATS metrics for this draft, and leaving other use cases for future work. With this change, would you be suggesting that we focus not just on CATS but also "other relevant computing metrics" outside the scope of CATS?

Reply:
I had misunderstood the original question. I agree with this comment, and pushed this change: https://github.com/VMatrix1900/draft-cats-metric-definition/pull/8/commits/5bba78918b1bcea6446906614f155de96b943296